examples/joke.ts

```diff
@@ -1,6 +1,11 @@
 import OpenAI from 'openai';
-import { assign, fromPromise, createActor, setup, log, raise } from 'xstate';
+import { assign, createActor, fromPromise, log, raise, setup } from 'xstate';
 import { createAgent } from '../src';
+import {
+  getRandomFunnyPhrase,
+  getRandomRatingPhrase,
+} from '../src/helpers/jokeHelper';
```

Comment:
Can we put this in the same file? It's not helpful for them to be in separate files (better to see everything in one place for examples)

Reply:
Done